### PR TITLE
Add Mocha test + AcceptStyleAndClassName HOC

### PR DESCRIPTION
**Pull request by WiNloSt**
- Modify `AcceptStyleAndClassName` to inline `className` and `style` prop instead of creating an extra div
- Add support for Mocha + Enzyme. And test `AcceptStyleAndClassName` HOC

### CODE REVIEW

File: src/test/setup.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require('babel-register')()
```
Comment (WiNloSt) [inline src/test/setup.js]: For any reviewer passing by. This file content is grab from [here](https://github.com/lelandrichardson/enzyme-example-mocha/blob/master/test/.setup.js)

**Comment by lukyth** [comment]
Should we group `AcceptStyleAndClassName.js` and its test to their own directory? In case there're many container component in the future. @WiNloSt

**Comment by WiNloSt** [comment]
Definitely!

**Comment by prontonatty** [comment]
Woo hooo! This card is completed by good engineering Practice :D

**Pull request merged**